package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "name": "findadoc",
+  "name": "@ourjapanlife/findadoc-frontend",
   "version": "1.0.0",
   "private": true,
+  "license": "BSD-3",
   "scripts": {
     "dev": "nuxt dev",
     "build": "nuxt build",
     "start": "nuxt start",
     "generate": "nuxt generate",
-    "netlify": "git submodule update --init --recursive --remote && nuxt generate",
+    "netlify": "nuxt generate",
     "lint": "eslint . --ext .js,.vue,.json && prettier --write **/*.{js,vue,json,scss,md}",
-    "postinstall": "husky install"
+    "prepare": "husky install"
```

Comment:
Apparently this is a much friendlier way to run the husky script, per their documentation. https://typicode.github.io/husky/#/?id=install